validators/register-validator-node.md

```diff
@@ -550,65 +575,106 @@ Signature: <your-signature>
 
 ```
 
-### Step 3.6: Link skale wallet address to your validator account using validator-cli
+### Step 3.7: Link skale wallet address to your validator account using validator-cli
 
-> Make sure you copied Node Address from STEP 3.4
+> Make sure you copied Node Address from STEP 3.5
 
 **Terminal Command:**
 
 ```bash
- sk-val validator link-address [NODE_ADDRESS] [SIGNATURE] --yes --pk-file ./pk.txt 
+ sk-val validator link-address [NODE_ADDRESS] [SIGNATURE] --yes --pk-file ./pk.txt
 ```
 
+Optional arguments:
+
+-   `--pk-file` - Path to file with private key (only for `software` wallet type)
+-   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
+-   `--yes` - Confirmation flag
+
+
 </Step>
 
 <Step id='four'>
 
-### Step 3.7: Send-Accept Delegation using validator-cli
+### Step 3.8: Send-Accept Delegation using validator-cli
 
-> Make sure you  already have at least 100 SKL tokens in your validator wallet for TestNet MSR is 100SKL tokens. 
+> Make sure you  already have at least 100 SKL tokens in your validator wallet for TestNet MSR is 100SKL tokens.
 
 **Terminal Command:**
 
 ```bash
  sk-val holder delegate --validator-id [Validator_ID] --amount 100 --delegation-period 3 --pk-file pk.txt --info "please accept delegation" --yes
 ```
+Required arguments:
+
+-   `--validator-id` - Validator id
+
+-   `--amount` - Delegation amount
+info)
+
+-   `--delegation-perion` - Delegation period (only 2 month allowed for
+now)
+
+-   `--info` - Delegation info
+
+Optional arguments:
+
+-   `--pk-file` - Path to file with private key (only for software wallet
+-   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
+type)
+
+-   `--yes` - Confirmation flag
+
 List your delegations
 
+**Terminal Command:**
+
 ```bash
-    sk-val validator delegations [VALIDATOR_ID] 
+    sk-val validator delegations [VALIDATOR_ID]
 ```
 
 You will see your pending delegation please get the delegation number and accept delegation
 
+**Terminal Command:**
+
 ```bash
-    sk-val validator accept-delegation --delegation-id [DELEGATION-ID] --pk-file pk.txt 
+    sk-val validator accept-delegation --delegation-id [DELEGATION-ID] --pk-file pk.txt
 ```
 
+Required arguments:
+
+-   `--delegation-id` - Delegation id to accept
+
+Optional arguments:
+
+-   `--pk-file` - Path to file with private key (only for software wallet type)
+-   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
```

Comment:
I wish the average was 1.25 default not 2 I'll approve but can we create a ticket for this?